basic/src/auth/AuthLayout.tsx

```diff
@@ -0,0 +1,9 @@
+export function AuthLayout({ children }: React.PropsWithChildren) {
+  return (
+    <div className="h-full w-full flex justify-center text-black">
+      <div className="w-full h-fit max-w-md bg-white px-8 py-10 rounded-lg mt-32 ">
+        <div>{children}</div>
+      </div>
+    </div>
```

Comment:
```suggestion
    <div className="flex justify-center">
      <div className="card mt-32 w-full max-w-md px-8 py-10">{children}</div>
    </div>
```

1. `w-full h-full` don't really do anything here for the container div
2. `text-black` is not really needed since you have the `text-*` class on the `App` container
3. You can probably reuse the `card` class here
4. No need for an extra `div` wrapper around the `children`